keps/sig-node/1665-grpc-probe/kep.yaml

```diff
@@ -0,0 +1,15 @@
+title: Add GRPC Probe
+kep-number: 1665
+authors:
+  - "@PxyUp"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: provisional
+creation-date: 2020-04-04
+reviewers:
+  - TBD
+approvers:
```

Comment:
I think we need a reviewer from sig-node and probably an approver, before this can be "implementable".